### PR TITLE
Added status checking utility, updated requirements.txt files

### PR DESCRIPTION
**Pull request by bendemeyer**

### CODE REVIEW

File: envs/python2/requirements.txt [hunk 1]
```diff
@@ -1,2 +1,4 @@
-hiredis==0.2.0
-redis==2.10.5
+hiredis>=0.2.0
+redis>=2.10.5
+mockredis
+tabulate>=0.7.5
```
Comment (bendemeyer) [inline envs/python2/requirements.txt]: This is a requirement for the status utility, but not for the actual circuit breaker module, so I added this to the requirements.txt files, but not to the setup.py

**Comment by bendemeyer** [comment]
Example useage and output:

```
$ python status.py -e "prod::http://pr-product-server.zoro.com/v1/categories/"
╒══════════════════════╤═════════╕
│ Health               │ healthy │
├──────────────────────┼─────────┤
│ History Cases        │ 1000    │
├──────────────────────┼─────────┤
│ History Successes    │ 994     │
├──────────────────────┼─────────┤
│ History Success Rate │ 99.4%   │
├──────────────────────┼─────────┤
│ Failure Threshold    │ 80.0%   │
╘══════════════════════╧═════════╛

$ python status.py -e "localhttp://0.0.0.0:8080/v1/recommendations/?"
╒═══════════════════════════╤═══════════╕
│ Health                    │ unhealthy │
├───────────────────────────┼───────────┤
│ Test Count                │ 2618      │
├───────────────────────────┼───────────┤
│ Test History Cases        │ 10        │
├───────────────────────────┼───────────┤
│ Test History Successes    │ 5         │
├───────────────────────────┼───────────┤
│ Test History Success Rate │ 50.0%     │
├───────────────────────────┼───────────┤
│ Recovery Threshold        │ 95.0%     │
╘═══════════════════════════╧═══════════╛
```

**Comment by 10chu** [comment]
👍

**Pull request merged**